cpp/include/cudf_test/column_utilities.hpp

```diff
@@ -199,8 +199,14 @@ template <>
 inline std::pair<thrust::host_vector<std::string>, std::vector<bitmask_type>> to_host(column_view c)
```

Comment:
IMO we should get rid of the `host_vector`s here in favor of `std::vector`. Also add an optional stream parameter and use cudaMemcpyAsync.

Reply:
This is the `cudf_test` column utilities which doesn't have any stream stuff in it.

Reply:
It should as part of adding stream support to the whole library.